### ISSUE TITLE
`gol query` outputs invalid csv

### ISSUE THREAD

**Issue by nlehuby** [clarisma/gol-tool#108]
The `gol query` command outputs invalid csv when the requested tags are not set on some OSM objets

For instance :
`gol query idf 'n[railway=subway_entrance]' -f=csv -t=name,ref,lat,lon > entrances.csv`

outputs
```
t,id,name,ref,lat,lon
N,7924971323,"Les Agnettes",48.9230355,2.2859444
N,1519417728,"La Plaine Stade de France",1,48.929194,2.3568467
N,8224750689,"Rue Danièle Casanova",2,48.9299347,2.3565682
N,8224750688,"Rue Danièle Casanova",2,48.9299633,2.3565688
(...)
```

The fist line has 5 columns while the other ones have 6, because the first OSM object does not have the wanted `ref` tag

(tested with 0.1.8 version)

**Comment by clarisma** [comment]
Thanks for submitting this!
`CsvFeaturePrinter` should output an empty value if the feature does not have a specified tag, in order to maintain the column structure.
A fix is coming shortly.

**Comment by clarisma** [comment]
Fix is in `dev`, will be released as part of v0.1.10
